src/db/model/stardust/mod.rs

```diff
@@ -5,3 +5,10 @@
 pub mod message;
 /// Module containing the milestone model.
 pub mod milestone;
+
+/// These are the names of the collections that we create in the database. At some point we could use a macro to make
+/// this list nicer.
+mod collection {
+    pub const MESSAGE_RECORDS: &str = "stardust_message_records";
```

Comment:
Any reason for the change from `_messages` and `_milestones` here?

Reply:
I wanted to be more explicit. For instance, the type that we store is `MessageRecord`, so I thought I'd bring these in line.